### PR TITLE
Fix typo heading

### PR DESCRIPTION
**Pull request by pierre-1**
Yada

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,6 @@
     }
   </style>
   <body>
-    <h1 class="some-heading">GIT WORKFLOW WORKSHOW</h1>
+    <h1 class="page-heading">GIT WORKFLOW WORKSHOP</h1>
```
Comment (tochman) [inline index.html:14]: Hmm... you also changed the class name. That is out of scope

Comment (pierre-1) [reply index.html:14]: Sorry about that, will be more careful!

Comment (tochman) [reply index.html:14]: Alright. You are forgiven.....
